watertap/unit_models/osmotically_assisted_reverse_osmosis_1D.py

```diff
@@ -0,0 +1,240 @@
+class OsmoticallyAssistedReverseOsmosis1DData(
+    OsmoticallyAssistedReverseOsmosisBaseData
+):
+    """
+    Standard 1D OARO Unit Model Class:
+    - one dimensional model
+    - steady state only
+    - single liquid phase only
+    """
+
+    CONFIG = CONFIG_Template()
+
+    _add_has_full_reporting(CONFIG)
+
+    def _process_config(self):
+        super()._process_config()
+        if self.config.transformation_method is useDefault:
+            _log.warning(
+                "Discretization method was "
+                "not specified for the "
+                "reverse osmosis module. "
+                "Defaulting to finite "
+                "difference method."
+            )
+            self.config.transformation_method = "dae.finite_difference"
+
+        if self.config.transformation_scheme is useDefault:
+            _log.warning(
+                "Discretization scheme was "
+                "not specified for the "
+                "reverse osmosis module."
+                "Defaulting to backward finite "
+                "difference."
+            )
+            self.config.transformation_scheme = "BACKWARD"
+
+    def _add_membrane_channels_and_geometry(self):
+
+        # Build membrane channel control volume
+        channel_kwargs = {
+            "dynamic": self.config.dynamic,
+            "has_holdup": self.config.has_holdup,
+            "area_definition": self.config.area_definition,
+            "property_package": self.config.property_package,
+            "property_package_args": self.config.property_package_args,
+            "transformation_method": self.config.transformation_method,
+            "transformation_scheme": self.config.transformation_scheme,
+            "finite_elements": self.config.finite_elements,
+            "collocation_points": self.config.collocation_points,
+        }
+
+        self.feed_side = MembraneChannel1DBlock(**channel_kwargs)
+        self.permeate_side = MembraneChannel1DBlock(**channel_kwargs)
+
+        self._add_length_and_width()
+        add_geometry_kwargs = {
+            "length_var": self.length,
+            "width_var": self.width,
+        }
+        self.feed_side.add_geometry(
+            flow_direction=FlowDirection.forward, **add_geometry_kwargs
+        )
+        self.permeate_side.add_geometry(
+            flow_direction=FlowDirection.backward, **add_geometry_kwargs
+        )
+        self._add_area(include_constraint=True)
+
+    def _add_deltaP(self, side):
+        if not isinstance(side, str):
+            raise TypeError(
+                f"{side} is not a string. Please provide a string for the side argument."
+            )
+
+        units_meta = self.config.property_package.get_metadata().get_derived_units
+        mem_side = getattr(self, side)
+        setattr(
+            mem_side,
+            "deltaP_stage",
+            Var(
+                self.flowsheet().config.time,
+                initialize=-1e5,
+                bounds=(-1e6, 0),
```

Comment:
I remember the bounds of pressure drop causing some issues in the multi-stage 0D OARO flowsheet. If my recollection is correct, should we remove these bounds? Even if it isn't correct, can we remove these bounds and still ensure (or even improve) model stability?

Reply:
Instead of relaxing the bounds to None, did you try 0?

Reply:
Yeah, my upper bound is set as 0

Reply:
Ahh yes, I had the bounds backwards in my head since the deltaP values are negative, my mistake